src/components/InsuranceAmountBlock/InsuranceAmountBlockInner.tsx

```diff
@@ -40,7 +40,7 @@ export const InsuranceAmountBlockInner = JSX<InsuranceAmountBlockInnerProps>(
     return (
       <section id={anchor} className={`min-w-full ${className}`}>
         <div className="relative mt-7 overflow-hidden">
-          {galleryLength > 1
+          {galleryLength !== 0
```

Comment:
Может быть так лучше?
```suggestion
          {galleryLength > 0
```

Reply:
Сделаю как описал Андрей  `const galleryLength = cards?.length ?? 0`